.github/CODEOWNERS

```diff
@@ -45,8 +46,7 @@
 /src/sinks/util/auto_concurrency @bruceg
 
 /src/sources/docker.rs @LucioFranco
-/src/sources/file/mod.rs @LucioFranco
-/src/sources/file/line_agg.rs @MOZGIII
+/src/sources/file.rs @LucioFranco
```

Comment:
This can probably be dropped, as he is no longer part of the team.

Reply:
True. I don't want this PR to introduce any changes like that though - to keed it simple and get in quickly.
I'd suggest we create a dedicated PR (or a series of dedicated PRs) to drop/reassign ownership.